src/lazy/text/buffer.rs

```diff
@@ -441,55 +466,53 @@ impl<'data> TextBufferView<'data> {
                 whitespace_and_then(alt((tag(","), peek(tag("}"))))),
             )
             .map(|invocation| Ok(Some(RawFieldExpr::MacroInvocation(invocation)))),
-            Self::match_struct_field_name_and_e_expression_1_1.map(
-                |((matched_name, name_span), invocation)| {
```

Comment:
🗺️ These closure parameters were consolidated, making this line shorter and causing `rustfmt` to reflow the lines that followed. 😞 This is not a logic change.